packages/app/src/__tests__/NexusPack.test.tsx

```diff
@@ -16,6 +16,10 @@ test('visualize NXdata group with "spectrum" interpretation', async () => {
   const tabs = await findVisSelectorTabs();
   expect(tabs).toHaveLength(1);
   expect(tabs[0]).toHaveTextContent(NexusVis.NxSpectrum);
+
+  expect(
+    await screen.findByRole('figure', { name: 'twoD_spectrum (arb. units)' }) // signal name + `units` attribute
+  ).toBeVisible();
```

Comment:
All those new expectations replace the NeXus `utils` unit tests for `getDatasetLabel` (which now requires fetching attribute values and can therefore no longer be unit tested easily).